src/app/about/officer-card.tsx

```diff
@@ -0,0 +1,56 @@
+import Image from "next/image";
+import styles from "./OfficerCard.module.css";
+
+type SocialLink = {
```

Comment:
[Suggestion]: Would recommend exporting this too since it's used in `OfficerCardProperties` which is exported.